src/components/molecules/selection.jsx

```diff
@@ -0,0 +1,51 @@
+import  React, { useState } from "react";
+import PropTypes from "prop-types";
+
+function Selection(props) {
+  const [selectionStyle, setSelectionStyle] = useState(disabled);
+
+  const disabled = {
+    border: "1px solid #a0a0a0",
+  };
+
+  const enabled = {
+    border: "1px solid #3b82f6",
+  };
+
+  const handleSelectionDivClick = (event) => {
+    event.target.checked
+      ? setSelectionStyle(enabled)
+      : setSelectionStyle(disabled);
+  };
+
+  return (
+    <div
+      className="selection-body"
+      onChangeCapture={handleSelectionDivClick}
+      style={selectionStyle}
+    >
+      <input
+        type="checkbox"
+        value={props.value}
+        name={props.name}
+        id={props.name}
+        className="selection-input"
+        onChange={props.selectionAction}
```

Comment:
I'd keep this consistently named as `onChange` - unless its doing something different

Reply:
fixed